src/components/extensionTopContent/ExtensionTopContent.jsx

```diff
@@ -0,0 +1,12 @@
+import "../../index.css";
+import GnbBox from "./top-contents/GnbBox";
```

Comment:
```suggestion
import GnbBox from "./top-contents/GnbBox";
```
GnbBox 컴포넌트의 Gnb가 무슨 뜻인가요? 혹시 약어라면 가독성이 좋도록 변경하는 것이 좋아보입니다.!